src/custom/pages/App/AppMod.tsx

```diff
@@ -105,18 +87,15 @@ function TopLevelModals() {
 }
 
 export default function App(props?: { children?: ReactNode }) {
-  const [bgBlur, setBgBlur] = useState(false)
   const location = useLocation()
-  useEffect(() => {
-    setBgBlur(location.pathname.length > 1 && location.pathname !== '/swap')
-  }, [location.pathname])
```

Comment:
Normally we don't modify mods (we would comment out the lines, so we still can use the comparer to check out the differences we applied. However in this case, you did it right, because this "AppMod" has many changes that break some of our rules, and specifically this lines were not part of the original code, so I think is fine to delete